### ISSUE TITLE
Implement new Gigya modal

### ISSUE THREAD

**Issue by stevepessah** [akcdotorg/akc.org#63]
https://akcorg.atlassian.net/browse/AKC2-408

Before doing this, we need to understand what happens to users who have signed up/logged in with other social accounts. (Related to ticket #131)

**Comment by kylejbertelsen** [comment]
@spaulenoff: please split this into 2 tickets: research the effect of this on pre-registered users; and implement the new Gigya modal. Estimate 2 hours for the first, and have Toshan make his own estimate for the second.

**Comment by spaulenoff** [comment]
Per @kylejbertelsen Comment, @toshan2014 Please provide an estimate for this and add this to your ticket queue. This ticket is in regards to implementation.

**Comment by kylejbertelsen** [comment]
BLOCKED by 131

**Comment by spaulenoff** [comment]
Removing from 3/31 Sprint
Removing from #132 
Need to research effects more

**Comment by spaulenoff** [comment]
We want to update the Mockup to show All six social media icons in two rows of 3 each.

This will allow us to keep all social media log in options. 
Will provide an updated mockup

**Comment by stevepessah** [comment]
@spaulenoff in the mockup, can you change the first field name to "Email Address or My AKC Username"

**Comment by spaulenoff** [comment]
@stevepessah To confirm, you just want to swap what is currently in the mockup?
![image](https://cloud.githubusercontent.com/assets/13648969/14252131/f6dbbf4c-fa53-11e5-9faf-44348e7ea8e4.png)

**Comment by toshan2014** [comment]
@spaulenoff The changes are ready to be pushed. Please confirm if we'll be using the new modal for both desktop and mobile.

**Comment by spaulenoff** [comment]
@stevepessah Please confirm if the new modal should be used on both desktop and mobile, or just on Desktop.

**Comment by toshan2014** [comment]
@stevepessah @spaulenoff @kylejbertelsen   We need to let the Marketplace team know that there will be changes to the login gigya screen as they use the same screen as akc.org. They prob will need to test out on their side too before we push to production.

**Comment by spaulenoff** [comment]
@toshan2014 Please use new modal on desktop and mobile

**Comment by toshan2014** [comment]
There are a total of 10 screens: 8 of which I've *_temporarily *_ put a link to in the Sign In menu for testing purposes + Login and Registration screens. Those 8 screens will obviously not process any data passed  upon Submit as their functionality depends on the status of the member's account but their field inputs can be tested.

Please QA: http://develop2.akc.org/

Time spent : 12h

**Comment by spaulenoff** [comment]
Thank you @toshan2014 Can we remove those links up top? Can you please also confirm if this is the modal that is also used on Marketplace?

**Comment by toshan2014** [comment]
As per Simon's comment, URLs to other screens have been removed. Please QA

Marketplace will have to test their modal on new screen : "Login-web-2-new "

**Comment by spaulenoff** [comment]
Verified on Dev2

**Comment by toshan2014** [comment]
Please qa on stage

**Comment by spaulenoff** [comment]
Looks good tome @stevepessah Please QA on Stage on desktop and mobile

**Comment by stevepessah** [comment]
Can we move the content up a bit so you can fully see the option to Sign Up?

@spaulenoff also check on iPhone 5 and tablet for those experiences.

![img_4749](https://cloud.githubusercontent.com/assets/1063028/14439674/5d227484-fffa-11e5-9d78-5062c0e632a1.PNG)

**Comment by spaulenoff** [comment]
@toshan2014 Please make changes per Steve's comment

**Comment by spaulenoff** [comment]
Could you also make sure the social buttons are center on iphone 5 and 5s
![iphone5 gigya](https://cloud.githubusercontent.com/assets/13648969/14440184/c43110fc-fffc-11e5-9c57-f5cd2563d258.png)

**Comment by toshan2014** [comment]
- Content moved up to reasonable amount so it does not look crammed. Everything is visible on the iphone6 and 6s without scroll.
- Social buttons alignment adjusted for iphone5 and 5s.

Also in order to avoid creating too many screen on Gigya and in an effort to have less redundant ones, "Login-web-2" contains the same things "Login-web-2-new". I will delete "Login-web-2-new" as soon as QA is done. On production we will use "Login-web-2" which is the one currently in use. Perhaps notify Marketplace of this. What it means simply is that:

**Use "Login-web-2" instead of "Login-web-2-new" for both testing and production.**
Please QA

**Comment by spaulenoff** [comment]
@toshan2014 Changes look good. On Mobile however, can we just add back in the line at the bottom of the sign in modal to show that it is separate from the rest of the page

**Comment by toshan2014** [comment]
@spaulenoff Border added back

**Comment by aalattili** [comment]
On android (Samsung S6)-Gigya sign up pop up, ‘Confirmation Password’ txt box isn’t aligned with ‘Password’ txt box since (*) is shifted under field title in portrait. (Android-Gigya Sign up-Alignment issue.png)
![android-gigya sign up-alignment issue](https://cloud.githubusercontent.com/assets/12289479/14529864/25c59108-025f-11e6-9f85-2a18fd147448.png)

**Comment by spaulenoff** [comment]
@toshan2014 Please make this update. Moving back to stage until this is finished.

**Comment by toshan2014** [comment]
@spaulenoff Are you OK using "Retype Password" instead of "Confirm Password". That will solve the problem as it's a matter of screen/text size. 

The other option would be to reduce the font-size. But then that would mean on mobile devices where it's working fine ( iphones ), the fonts will be smaller.

**Comment by spaulenoff** [comment]
@stevepessah What do you think per Toshan's comment to the issue Aspire brought up on Android? Another option would be to abbreviate Password to PW. Or we could just leave it how it is as this is only an issue on the Android Sign Up screen.

I personally Retype could be a good fallback option as it conveys the same message and it will fit on one line instead of two how it is in the above screenshot.

Please advise on how you would like to proceed.

**Comment by spaulenoff** [comment]
@toshan2014 Confirmed ok to use Retype

**Comment by toshan2014** [comment]
Fixed
![13020066_10154030472615761_691201685_n png](https://cloud.githubusercontent.com/assets/6847220/14537398/afb93e20-022b-11e6-8a88-664b643b407c.jpg)

**Comment by aalattili** [comment]
@spaulenoff , Gigya login in "Learn" pages has different design compared to rest of website,
List of issues in Gigya login in learn pages:
1- Dialog top/caption section is dark violet-colored
2- Background color/ 'OR' appears inside a box with color lighter than background
3- Font color & style
4- Button size (Dimensions)[Learn login button: 180 px \* 56 px, in other pages 180 px \* 40 px)
5- Boarders of text boxes are black-colored while grey-colored in rest of website 
Please refer to attached screenshot for more comparing details (Gigya login new design-Bug in learn pages.png)

_Note:_ This issue is applicable on all desktop browsers and mobile web for both sign up/in gigya dialog.

![gigya login new design-bug in learn pages](https://cloud.githubusercontent.com/assets/12289479/14587017/dcb45052-04b2-11e6-85db-e9b70de5418a.png)

**Comment by aalattili** [comment]
@spaulenoff, Opening gigya login dialog will cause articles behind Gigya login dialog to hide/ diappear.
Note: This issue is applicable on mobile web for both sign up/in Gigya dialog.

![learn pages-opening login dialog will cause all articles beneath it to hide](https://cloud.githubusercontent.com/assets/12289479/14587128/bfa9ede8-04b5-11e6-81c3-693190eeb06f.PNG)

**Comment by toshan2014** [comment]
pf:
http://stage-ng.akc.org/learn/
http://stage-ng.akc.org/

**Comment by spaulenoff** [comment]
Verified on Prod

**Comment by toshan2014** [comment]
Total Time spent : 15h
